### PR TITLE
[PLAT-148] Add max-processor-accumulated-records yaml & xml examples

### PR DESCRIPTION
**Pull request by gierlachg**

### CODE REVIEW

**Comment by gierlachg** [comment]
run-lab-run

File: site/docs/design-docs/019-memory-management.md [hunk 1]
```diff
@@ -65,33 +65,51 @@ it if it turns out that current solution is not enough.
...
-    // ...
 }
 ```
 
+<!--YAML-->
+
+```yaml
+  jet:
+    instance:
+      max-processor-accumulated-records: 100
```
Comment (viliam-durina) [inline site/docs/design-docs/019-memory-management.md:87]: This is a bit low for an example.

**Review by viliam-durina — APPROVED** [review]

**Comment by Holmistr** [comment]
From the SQL user perspective, it would make much more sense to have it under `<sql>` element. Is there a way how to distinguish between SQL queries and "normal" Jet jobs? Would it make sense to have the configuration possibly separate for these use cases?

**Comment by gierlachg** [comment]
SQL jobs are no different than _regular_ jobs. Thus, we decided to implement the limit for _both_. Currently, there is no way to distinguish between the two. I'm not sure if it would make sense to have 2 configuration options - the behaviour is exactly the same, moreover you can always override the member value with the job one.

**Review by gurbuzali — APPROVED** [review]

File: hazelcast/src/main/resources/hazelcast-full-example.xml [hunk 2]
```diff
@@ -3704,6 +3721,7 @@
             <backup-count>1</backup-count>
             <scale-up-delay-millis>10000</scale-up-delay-millis>
             <lossless-restart-enabled>false</lossless-restart-enabled>
+            <max-processor-accumulated-records>1000000000</max-processor-accumulated-records>
```
Comment (viliam-durina) [inline hazelcast/src/main/resources/hazelcast-full-example.xml:3724]: This is quite a lot... Why not putting 1M?

Comment (gierlachg) [reply hazelcast/src/main/resources/hazelcast-full-example.xml:3724]: We want to provide an example and keep backward compatibility (not having it at all), I think 1M can be quite easily reached - I would say 1G is a fair compromise.

**Review by Holmistr — APPROVED** [review]

**Pull request merged**